.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+ADO card: ☑️ [AB#XXXX](https://dev.azure.com/S72/SHIFT72/_boards/board/t/Delivery%20team/Stories/?workitem=XXXX)
+
+Elab notes/AC: 📃 [Google Docs](https://docs.google.com)
+
+Designs: 
+- 📱 [Mobile](http://www.zeplin.io/????)
+- 💊 [Tablet](http://www.zeplin.io/????)
+- 🖥 [Desktop](http://www.zeplin.io/????)
+
+## Description of work
+This is a description of the feature or bug, including any detail not in the ADO card that is required to understand the work.
```

Comment:
I'm going to suggest that ADO cards are usually a bit lacking, we should be making it easy for reviewers and require a decent description here.